pulumi/grapl/__main__.py

```diff
@@ -198,11 +307,47 @@ def main() -> None:
 
         # This does not use a custom Provider since it will use either a consul:address set in the config or default to
         # http://localhost:8500. This also applies to the NomadJobs defined for LOCAL_GRAPL.
+        bootstrap = ConsulAclBootstrap(
+            name="consul-acl-bootstrap",
+            consul_address=pulumi.Config("consul").get("address")
+            or "http://localhost:8500",
+            secret_token=pulumi.Config("consul").get("token"),
```

Comment:
Should this be `Config(...).get_secret(...)` instead? You may not need the `additional_secret_outputs` option in that case.

Reply:
So... if we provide a token, then we've basically already bootstrapped, correct? In that case, we just use that master token to do all the other provisioning.

Isn't the bootstrap process more of a cluster initiation step? It seems odd that it's baked into this "client" project. Also, the token produced has unlimited capabilities, which suggests that it should be protected centrally, and that we should probably have another mechanism by which to supply a more tightly-scoped "Grapl-only" token for this purpose.